src/layouts/blog.js

```diff
@@ -1,20 +1,23 @@
 import React, { Fragment } from 'react'
 
 import Banner from '../components/banner'
-import BlogNav from '../components/blog/blog-navbar'
+import Navbar from '../components/navbar-light'
+import BlogHero from '../components/blog/blog-hero'
+import PostNav from '../components/blog/post-nav'
 import Footer from '../components/footer'
 
-
-export default ({ children }) => (
+export default () => (
   <Fragment>
-    {/* <Banner /> */}
-    <header className="header blog-header">
-      <BlogNav />
-      <div className="container">
-        <h1 className="blog-title">Storj Blog</h1>
+    <Banner />
+    <header className='header blog-header'>
+      <Navbar />
+      <div className='container'>
+        <BlogHero />
       </div>
     </header>
-
+    <section id='blog-list' className='container-fluid'>
+      <PostNav />
```

Comment:
Is the grid portion of the blog page not built out yet?
<img width="1246" alt="Screen Shot 2019-08-15 at 12 44 11 PM" src="https://user-images.githubusercontent.com/22300958/63110958-950b0500-bf5a-11e9-923d-bb493fad997c.png">
This is what I see on localhost so far

Reply:
Correct. I'm gonna be building that out tonight.